src/models/pressure-system.ts

```diff
@@ -18,13 +18,19 @@ export interface IPressureSystemOptions {
 const minLat = 15;
 
 const minDistToOtherSystems = (center: ICoordinates, otherSystems: PressureSystem[]) => {
-  const dists = otherSystems.map(ps => distanceTo(ps.center, center));
+  const dists = otherSystems.map(ps => distanceTo(
+    {lat: ps.center.lat, lon: ps.center.lng},
+    {lat: center.lat, lon: center.lng}
+  ));
```

Comment:
I found this change and the others like it in this file to be necessary for reconciling type differences between `ICoordinates` and `geolocation-utils`. `ICoordinates` uses `lng` for longitude whereas `geolocation-utils` uses `lon`. I'm not sure exactly why that wasn't an issue before, but it must have something to do with the way I modified the definitions of the pressure systems in `src/config.ts`. It seems to me that it's a good change to make, although I wonder if it'd be best to modify `ICoordinates` to use `lon` instead of `lng`. I'm wary of making that change at this time, however.